src/hooks/useMe.tsx

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from 'react-query';
+import { useActiveAccount } from './useActiveAccount';
+import { useHttpClient } from './useHttpClient';
+
+export interface Me {
+  patientId: string;
+}
+
+interface MeResponse {
+  resourceType: 'Bundle';
+  entry: [
+    {
+      resource: {
+        id: string;
+      };
+    },
+  ];
+}
+
+export function useMe() {
+  const { accountHeaders } = useActiveAccount();
+  const { httpClient } = useHttpClient();
+
+  return useQuery(
+    'fhir/dstu3/$me',
+    () =>
+      httpClient
+        .get<MeResponse>('/v1/fhir/dstu3/$me', { headers: accountHeaders })
+        .then((res) => ({
+          patientId: res.data.entry?.[0]?.resource?.id,
```

Comment:
Seems good for now, I imagine we will want this to support multiple patient mappings in the future.

Reply:
Yep, agreed.  What we have now is "account selection", but it'd be great to make that "account+project" selection.